src/main/java/net/sf/jabref/external/push/PushToApplicationButton.java

```diff
@@ -56,9 +54,10 @@
     private final JPopupMenu optPopup = new JPopupMenu();
     private final JMenuItem settings = new JMenuItem(Localization.lang("Settings"));
 
-    public PushToApplicationButton(JabRefFrame frame, List<PushToApplication> pushActions) {
-        this.frame = frame;
-        this.pushActions = pushActions;
+
+    public PushToApplicationButton(JabRefFrame frameJR, List<PushToApplication> pushActionsList) {
+        frame = frameJR;
+        pushActions = pushActionsList;
```

Comment:
Hm, I would leave it for setting something in the constructor/setter. What do the other @JabRef/developers think?